coalib/output/ConsoleInteraction.py

```diff
@@ -529,7 +529,10 @@ def acquire_settings(log_printer, settings_names_dict, section):
     for setting_name, arr in sorted(settings_names_dict.items(),
                                     key=lambda x: (join_names(x[1][1:]), x[0])):
         value = require_setting(setting_name, arr, section)
-        result.update({setting_name: value} if value is not None else {})
+        while value is None:
+            print('You have not entered any value.')
```

Comment:
just reask, I wouldn't print this^^ Most linux tools don't print such a message either^^